src/internal/bind-to-namespace.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Bind all methods of a class to the class instance.
+ *
+ * @param moduleClass The class to bind the methods of.
+ * @param that The class instance to bind the methods to.
```

Comment:
```suggestion
 * @param that The class instance of which the methods are to be bound to itself.
```

Not sure how to word this better.